integration/presto/pom.xml

```diff
@@ -688,5 +675,105 @@
         <maven.test.skip>true</maven.test.skip>
       </properties>
     </profile>
+    <profile>
+      <id>prestodb</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
Seems there are two `activeByDefault`

Reply:
Now user need to select profile when building for presto integration, so CI only will test for prestosql integration, right?

Reply:
By default sprak2.3 and prestosql is active. Hence intentionally made 2 active profiles. 
If user needs prestodb he needs to specify the profile, similar to spark2.4